### ISSUE TITLE
Wide rules should be considered invalid if there is no domain restriction

### ISSUE THREAD

**Issue by ameshkov** [AdguardTeam/CoreLibs#11]
Invalid wide rules:
```
$websocket
*$script
||$domain=~example.org
|*$xmlhttprequest
```

**Comment by hmage** [comment]
Fixed in commit aec2a865c783a45a0a9bf245eab7e616d19e209c

**Comment by TPS** [comment]
Just to clarify, did this exclude `$app=`, as well?  Or is that irrelevant _here_, & be handled by the mobile apps? (Unless you're planning to extend this into an _any_ OS-wide program, in which case `$app=` will be relevant *every*place.)  E.g., I could see doing `$script,app=com.example.GoodContentKookyScripts`.  Would that still be allowed?

**Comment by ameshkov** [comment]
@TPS great catch! Yeah, I agree, we should also allow those with $app modifier

**Comment by hmage** [comment]
Fixed in commit caf1a8a224fd3632ca9bc2a1d35ffcf05673cdc0. If rule has apps in $app modifier, it's never considered wide.
